### ISSUE TITLE
How to use tab with colorful icon

### ISSUE THREAD

**Issue by duongnv1996** [aurelhubert/ahbottomnavigation#314]
I have colourful icon with many different colours. That I wanna make this to icon tab. Not only two colors. So I can find out how to set with the original colour icon. 
Could you please help me, guy!
Thank in advance!

**Comment by TrungNguyen1208** [comment]
me too :+1:

**Comment by duongnv1996** [comment]
I handled this by implementing library and custom in java file. If you need the code, Feel free to contact me via mail (duongnv1996@gmail.com).  Because I don't have source code right now, so just ping me if you need.

**Comment by TrungNguyen1208** [comment]
yeah! my email: thanhtrungnguyen1208@gmail.com . Thank you for help me :+1:

**Comment by alpamys-qanybet** [comment]
@duongnv1996, I also need it. Best regards, email: [alpamys.kanibetov@gmail.com](mailto:alpamys.kanibetov@gmail.com)
